src/main/java/io/debezium/connector/spanner/db/dao/SchemaDao.java

```diff
@@ -77,21 +82,33 @@ public ChangeStreamSchema getStream(Timestamp timestamp, String streamName) {
         return exist ? builder.build() : null;
     }
 
-    private ResultSet readTablesInfo(ReadOnlyTransaction tx, Collection<String> tables) {
+    private ResultSet readColumnsInfo(ReadOnlyTransaction tx, Collection<String> tables) {
+        Statement statement = Statement.newBuilder("SELECT" +
+                "  table_name," +
+                "  column_name," +
+                "  spanner_type," +
+                "  ordinal_position," +
+                "  IF(is_nullable = 'YES', true, false) AS is_nullable\n" +
+                "FROM" +
+                "  information_schema.COLUMNS \n" +
+                "WHERE" +
+                "  table_catalog = ''" +
+                "  AND table_schema = ''" +
+                (tables == null ? "" : "  AND table_name in UNNEST(@tables)"))
+                .bind("tables")
+                .toStringArray(tables)
+                .build();
+        return tx.executeQuery(statement);
+    }
+
+    private ResultSet readPrimaryColumns(ReadOnlyTransaction tx, Collection<String> tables) {
```

Comment:
instead of saying "primaryColumns", should we say "primaryKeys"? Same everywhere else.

Reply:
Yeah sure.